C++/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include <QCoreApplication>
+#include <iostream>
+#include <linkedlist.h>
+
+using namespace std;
+
+int main()
+{
+
+    LinkedList<int> *list = new LinkedList<int>();
+
+    for (int var = 0; var < 5; ++var)
+    {
+        int init = rand()/10e7;
+        std::cout<<"inside main.."<<init<<std::endl;
+        list->Add(init);
+    }
+
+    list->PrintList();
+    list->RemoveAt(3);
+    list->PrintList();
+    list->Reverse();
+    list->PrintList();
+
+    delete list;
```

Comment:
Since you created this list using your own memory, you are now responsible in cleaning it. Always try to avoid memory creation manually using new() .